readme.md

```diff
@@ -76,7 +76,7 @@ String methods are discussed in [Chapter 8](#chapter8).
 
 * _Inside_ a function, the var statement creates variables local to that function
 * _switch, while, for_ and _do_ statements can have an optional **label**  which can be used with `break` and `continue` to provide more precise [control over exactly which statement to break or continue](http://www.tutorialspoint.com/cgi-bin/practice.cgi?file=javascript_19). Format: `labelname: statement` and then `continue labelname;`
-* A _block_ (statements with {}) **do not create new scope** - the variables are local to the function, not the block. So **always define variables at the top of the function** and not inside the blocks
+* ES2015 presents two new keywords for declaring variables, _let_ and _const_. Whereas the _var_ keyword is function scoped (the variables are local to the function), _let_ and _const_ are both block scoped, which means they are local to any statement with {}. 
```

Comment:
@jgmcalpine consider using "back ticks" for code e.g: `let` and `const` 
so it's clear these are `code` keywords
see: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

+ Windows: https://superuser.com/questions/254076/how-do-i-type-the-tick-and-backtick-characters-on-windows
+ Mac: https://apple.stackexchange.com/questions/69820/typing-a-backtick-on-mac-os-x-snow-leopard